### PR TITLE
Create Get-SPOTenantRenameStatus

### PR DESCRIPTION
**Pull request by WayneEwington**
Documenting the new PowerShell cmdlets to support SPO Tenant Renames.

Please include this PR with the other files:
* Start-SPOTenantRename
* Get-SPOTenantRenameStatus
* Stop-SPOTenantRename

### CODE REVIEW

**Review by tseward — APPROVED** [review]

**Comment by WayneEwington** [comment]
@yogkumgit  How long is it expected for JohanFreelancer9 to do the review as this article needs to be published. (Also note that I can't @mention him.) I also can't resolve the conflicts

**Comment by yogkumgit** [comment]
@tiburd Dario and Johan cannot be tagged or assigned/review, please help. Thanks

**Review by get-itips — CHANGES_REQUESTED** [review]
Please rename the file, it is missing the .md extension

File: sharepoint/sharepoint-ps/sharepoint-online/Get-SPOTenantRenameStatus [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+## DESCRIPTION
+
+This command gets the status of the job to rename the SharePoint domain name for your organization.
+
+## EXAMPLES
+
+### EXAMPLE 1
+
+```powershell
+Gett-SPOTenantRenameStatus
```
Comment (get-itips) [inline sharepoint/sharepoint-ps/sharepoint-online/Get-SPOTenantRenameStatus:38]:
```suggestion
Get-SPOTenantRenameStatus
```

File: sharepoint/sharepoint-ps/sharepoint-online/Get-SPOTenantRenameStatus [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+
+## RELATED LINKS
+
+[Getting started with SharePoint Online Management Shell](https://docs.microsoft.com/powershell/sharepoint/sharepoint-online/connect-sharepoint-online?view=sharepoint-ps)
+
+[Rename your SharePoint domain](https://aka.ms/SPOTenantRename)
+
+[Start-SPOTenantRename](Start-TenantRename.md)
+
+[Get-SPOSiteRenameState](Get-TenantSiteState.md)
```
Comment (get-itips) [inline sharepoint/sharepoint-ps/sharepoint-online/Get-SPOTenantRenameStatus:57]: Is this the link intended? the link description differs from the actual file name.

File: sharepoint/sharepoint-ps/sharepoint-online/Get-SPOTenantRenameStatus [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+author: WayneEwington
+ms.author: waynewin
+ms.reviewer:
+---
+
+# Get-SPOTenantRenameStatus
+
+## SYNOPSIS
+
+ [!IMPORTANT]
```
Comment (dariomws) [inline sharepoint/sharepoint-ps/sharepoint-online/Get-SPOTenantRenameStatus:17]:
```suggestion
> [!IMPORTANT]
```

Same as https://github.com/MicrosoftDocs/OfficeDocs-SharePoint-PowerShell/pull/11.

File: sharepoint/sharepoint-ps/sharepoint-online/Get-SPOTenantRenameStatus [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+
+This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable, -InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose, -WarningAction, and -WarningVariable. For more information, see [about_CommonParameters](https://go.microsoft.com/fwlink/?LinkID=113216).
+
+## RELATED LINKS
+
+[Getting started with SharePoint Online Management Shell](https://docs.microsoft.com/powershell/sharepoint/sharepoint-online/connect-sharepoint-online?view=sharepoint-ps)
+
+[Rename your SharePoint domain](https://aka.ms/SPOTenantRename)
+
+[Start-SPOTenantRename](Start-TenantRename.md)
```
Comment (dariomws) [inline sharepoint/sharepoint-ps/sharepoint-online/Get-SPOTenantRenameStatus:55]:
```suggestion
[Start-SPOTenantRename](Start-SPOTenantRename.md)
```

**Comment by yogkumgit** [comment]
@WayneEwington Please implement the suggested changes, if you agree. Thanks

**Comment by WayneEwington** [comment]
Renaming the file to include .md has "confused" this Pull Request, so closing this pull request and creating a new one.

**Pull request closed without merging**